### DISCUSSION TITLE
Support for Atom feeds in News Panel

### DISCUSSION THREAD

**Discussion by aengusrooneygrafana** [grafana/grafana#38162] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Support for Atom feeds in News Panel.  Some of our customers would like to implement internal news feeds based on Atom.   who have requested support for Atom feeds in news panel.  Workarounds on client side are not very robust (RSS to Atom converters), supporting Atom within news panel itself would be a good addition that would benefit our community.

**Why is this needed**:

Currently not supported, as pointed out in the panel itself 

![image](https://user-images.githubusercontent.com/54800912/96101129-6b8a9800-0ed5-11eb-905c-1126982301d8.png)

**Comment by MichaelTendoSsemwanga** [comment]
+1

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by stanier** [comment]
+1

**Comment by kaydelaney** [comment]
Merged a PR adding Atom support here: https://github.com/grafana/grafana/pull/45390
